configuration/configure-spdockit.md

```diff
@@ -20,6 +20,17 @@ This article explains how to configure the SPDocKit database and service before
     * Automatic Subscriptions and Alerts
     * Index Reorganization
     * Automatic Rules and Queries
-4. On the __Service Configuration__ wizard page click __Next__ to complete the Configuration Wizard and apply the changes.
+
+    If you installed a SPDocKit Insights Agent and wish to setup SPDocKit Insights Service, you can also check __Enable SPDocKit Insights Service.__  
+    As of version 7.2 there is only one Configuration Wizard for setting up the database, SPDocKit and SPDocKit Insights Service. Also note that now you must use the __same service account for both SPDocKit and SPDocKit Insights Service.__ 
```

Comment:
This note about the same account is only valid in case that agent and console are installed on the same server. Otherwise, you can use a differnt account.

Reply:
Not sure I follow? New CW demands that you use the same service account. If all you have is the event viewer then you don't need a service account at all. Only case i can remember where you could use a different account, is if you have more than one agent.